### DISCUSSION TITLE
No longer able to log in on firefly after latest update

### DISCUSSION THREAD

**Discussion by Mcblazeallot** [iotaledger/firefly#3093] [Help & Support]
Went to check out my new ASMB staking rewards, and it says I need to create an account instead of being able to log in. Is anyone else experiencing this issue?

**Comment by MiKeF05** [comment]
This happened to some users when the entry in the keychain of the OS has been lost. You can try to recover your wallet either with your stronghold backup + password or your 24 words. If you use a ledger device then you can recover your wallet with the ledger.
